db/seeds.rb

```diff
@@ -0,0 +1,13 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+
+
+Todo.destroy_all
+
+laundry = Todo.create(content:"Laundry", is_completed: false)
+dishes = Todo.create(content:"Dishes", is_completed: false)
```

Comment:
These commands look good, but if they weren't working then that might be because you don't have a model file.